extensions/wikia/SiteWideMessages/SpecialSiteWideMessages_body.php

```diff
@@ -1138,7 +1138,7 @@ static function getAllUserMessages($user, $dismissLink = true, $formatted = true
 			. ' WHERE msg_removed = ' . MSG_REMOVED_NO
 			. ' AND msg_mode = ' . MSG_MODE_ALL
 			. ' AND (msg_expire IS NULL OR msg_expire > ' . $DB->AddQuotes(date('Y-m-d H:i:s')) . ')'
-			. " AND " . MSG_TEXT_DB . ".msg_date > '{$user->mRegistration}'"	//fix for ticket #2624
+			. " AND " . MSG_TEXT_DB . ".msg_date > '".$user->getRegistration()."'"	//fix for ticket #2624
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)